docs/en/compute/Spark.md

```diff
@@ -140,17 +141,39 @@ After Alluxio 1.8 (not included), users can encode the Zookeeper service address
 inside an Alluxio URI (see [details](#access-data-from-alluxio-in-ha-mode)).
 In this way, it requires no extra setup for Spark configuration.
 
-### Customize Alluxio User Properties for Spark Jobs
+### Customize Alluxio User Properties for Individual Spark Jobs
 
-To customize Alluxio client-side properties in a Spark job, see
-[how to configure Spark Jobs](Configuration-Settings.html#spark-jobs).
+Spark users can use pass JVM system properties to Spark jobs by adding `"-Dproperty=value"` to
+`spark.executor.extraJavaOptions` for Spark executors and `spark.driver.extraJavaOptions` for
+Spark drivers. For example, to submit a Spark job with the write `CACHE_THROUGH` when writing to
+ Alluxio:
+
+```bash
+$ spark-submit \
+--conf 'spark.driver.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+--conf 'spark.executor.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+...
+```
+
+Note that, in client mode you need set `--driver-java-options "-Dalluxio.user.file.writetype.default=CACHE_THROUGH"` instead of
+`--conf spark.driver.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH` (see [explanation](https://spark.apache.org/docs/latest/configuration.html)).
+
+
+In the Spark Shell, this can be achieved by:
```

Comment:
Will this work for a client driver mode? (It will probably work for a cluster driver mode)